src/pages/patientView/PatientViewPage.tsx

```diff
@@ -883,6 +883,9 @@ export default class PatientViewPage extends React.Component<
                                         this.patientViewPageStore
                                             .sampleToMutationGenePanelId
                                             .isComplete &&
+                                        this.patientViewPageStore
+                                            .genePanelIdToEntrezGeneIds
```

Comment:
mutation data used for the mutation table depends on `genePanelIdToEntrezGeneIds` data (https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/clinicalInformation/PatientViewPageStore.ts#L1253)